dali/pipeline/data/tensor_vector.h

```diff
@@ -373,19 +492,150 @@ class DLL_PUBLIC TensorVector {
    */
   DLL_PUBLIC Tensor<Backend> AsTensor();
 
+  /**
+   * @name Metadata access
+   * @{
+   */
+  /**
+   * @brief Set uniform layout for all samples in the batch
+   */
+  void SetLayout(const TensorLayout &layout);
+
+  /**
+   * @brief Get the layout of the sample in the batch.
+   */
+  TensorLayout GetLayout() const;
+
+  /**
+   * @brief Set uniform layout for all samples in the batch
```

Comment:
I guess SetSkipSample does not really do that.

Reply:
done